src/main/java/de/tum/in/www1/artemis/repository/StudentExamRepository.java

```diff
@@ -389,8 +391,10 @@ default List<StudentExam> generateStudentExams(final Exam exam) {
         // https://jira.spring.io/browse/DATAJPA-1367 deleteInBatch does not work, because it does not cascade the deletion of existing exam sessions, therefore use deleteAll
         deleteAll(existingStudentExams);
 
+        Set<User> users = exam.getExamUsers().stream().map(ExamUser::getUser).collect(Collectors.toCollection(HashSet::new));
```

Comment:
Why not simply `.collect(Collectors.toSet())`?

Reply:
I changed to `.collect(Collectors.toSet())` :+1:

Reply:
Still unresolved.